funcx_container_service/application.py

```diff
@@ -0,0 +1,12 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route("/environment")
```

Comment:
I prefer the organization provided by [flask-restful library](https://flask-restful.readthedocs.io/en/latest/quickstart.html) where each REST endpoint is in its own class

Reply:
I also like rest URIs to be plurals. POST /environments pushes a new one, GET /environments gets all, GET /environments/:id gets one in particular.

Reply:
done